### DISCUSSION TITLE
Does Video DownloadHelper work for all YouTube videos?

### DISCUSSION THREAD

**Discussion by FarneyBlather** [aclap-dev/video-downloadhelper#330] [Help and Q&A]
### Your question

Some YouTube videos employ security schemes, such as encrypted streams or streams broken into components. Does this extension download all YouTube videos?

It is difficult to tell if a video is downloading or not, which is a serious UI flaw. There is little indication when things do not work properly.

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: false, browser: firefox
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:124.0) Gecko/20100101 Firefox/124.0
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\*****i",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
It should work on most user generated videos. It may have trouble on certain age restricted videos. 
There was a problem for users signed into youtube premium, downloading wouldn't work unless signed out.
Don't know if this was ever solved, I'm just another user.
